README.md.tt

```diff
@@ -97,13 +97,8 @@ docker-compose -f docker-compose.test.yml run test bin/bundle exec spec [rspec-p
 ```
 
 ### Automated Code Review Setup
+- Add a bot (i.e. `team-nimblehq`) to this repository or to the organization. The bot requires permission level “Write” to be able to set a PR’s status.
 
-* Make sure that the config file `pronto.yml` for pronto has been added as a configuration file.
-
-* In the CI build script, add the following command:
-
-```sh
-if ([ $BRANCH_NAME != 'master' ] && [ $BRANCH_NAME != 'development' ]); then \
-  (echo "Running pronto"; bundle exec pronto run -f bitbucket_pr -c origin/development);  else \ (echo "Escaping pronto"); \
-fi
-```
+- Create a [Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
+from bot account with `public_repo` scope, and set it as `DANGER_GITHUB_API_TOKEN` secret on Semaphore Environment
+Settings.
```

Comment:
@rafayetn small thing, seem the `Settings.` is on a new line? Could you double-check?

Reply:
Actually the word is in the different line as a line break in the text but it will adjust as per the screen size.

![image](https://user-images.githubusercontent.com/67670434/90600463-67fdcc00-e218-11ea-8007-331efd80b48a.png)

Reply:
Might I know the purpose to keep the `Setting.` at another line @rafayetn 🤔 ?

Reply:
Well, there is no significant purpose. It's just after 120 characters when a new word is written it goes to the next line to maintain readability. It's also as per the `.rubocop.yml` that we use to maintain 120 characters. Rubocop is integrated into my IDE so when I write something it checks for issues.

Reply:
Was the rubocop applied to the `md.tt` file 🤔.

IMO I would prefer to keep the Setting as the same line with another one @rafayetn, it looks really weird, what do you think @noor-ahmed @byhbt @junan

Reply:
Seems legit to me @rafayetn 👍

Reply:
And if the rubocop applied to that file, it should format another line too 

<img width="1300" alt="Pasted_Image_8_19_20__14_04" src="https://user-images.githubusercontent.com/11751745/90603082-ebbdb580-e224-11ea-8754-6d33096f4416.png">

@rafayetn

Reply:
Yeah... I usually write `ReadMe` in one line regardless of length. Sometimes IDE changes it as per settings as @rafayetn said. So we can make it in one line. ✌️

Reply:
I understand your concern. I just changed it 🚀